content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -86,34 +85,114 @@ With the microservice UUID from the last step, let's change our routes so that o
+      "config": "{ \"maxWindowSize\": 10 }",
+      "catalogItemId": 105,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+    }
+  ]
+}
+```
 
-```sh
-# Note the semicolon between the two UUIDs!
-iofog-controller microservice route-remove \
-  --route <sensors_uuid>:<api_uuid>
+After finding those two UUIDs in the list, provide the Sensors UUID and API UUID separated by a semicolon.
+```bash
+docker exec -ti iofog-controller \
+    iofog-controller microservice route-remove \
+        --route PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb:NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3
 ```
 
 Now we need to place two new routes: one from the Sensors to Moving Average, and another from Moving Average to the REST API; this places our new microservice in between them.
 
-```sh
-# Sensors -> Moving Average
-iofog-controller microservice route-create \
-  --route <sensors_uuid>:<moving_average_uuid>
+```bash
+docker exec -ti iofog-controller \
+    iofog-controller microservice route-create \
+        --route PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb:ydmtBFxJhxvVgLKfM2qLPj4KtcVgdg23
+        
+docker exec -ti iofog-controller \
+    iofog-controller microservice route-create \
+        --route ydmtBFxJhxvVgLKfM2qLPj4KtcVgdg23:NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3
+```
 
-# Moving Average -> REST API
-iofog-controller microservice route-create \
-  --route <moving_average_uuid>:<rest_api_uuid>
+The new configuration of microservices should look like this:
+
+```json
+{
+  "microservices": [
+    {
+      "name": "Sensors",
+      "uuid": "PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb",
+      "catalogItemId": 102,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+      "routes": [
+        "ydmtBFxJhxvVgLKfM2qLPj4KtcVgdg23"
+      ]
+    },
+    {
+      "name": "Rest API",
+      "uuid": "NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3",
+      "catalogItemId": 103,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+    },
+    {
+      "name": "Freeboard",
+      "uuid": "BrKHZf9PTcT6yjKcrpnRVBcYPFqxqXxb",
+      "catalogItemId": 104,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+    },
+    {
+      "name": "Moving Average",
+      "uuid": "ydmtBFxJhxvVgLKfM2qLPj4KtcVgdg23",
+      "config": "{ \"maxWindowSize\": 10 }",
+      "catalogItemId": 105,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+      "routes": [
+        "NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3"
+      ]
+    }
+  ]
+}
 ```
 
-Finally, for the moment of truth. Let's first try a curl request to our REST API:
+<aside class="notifications danger">
+  <h3><img src="/images/icos/ico-danger.svg" alt=""> Known issue - Agent deletes moving average image!</h3>
+  <p>When you update the routes for the moving average microservice, ioFog agent re-creates the containers, but due to a known issue, also deletes the underlying image. In order for the moving average image to start correctly, you need to build it again.</p>
+</aside>
+
+Finally, for the moment of truth. Let's first try a curl request to our REST API.
```

Comment:
THIS IS EXCITING! LET'S USE AN EXCLAMATION POINT!

Finally, ~~for~~ the moment of truth**!**